### ISSUE TITLE
Decouple the rpc connection and graph ql client on app start

### ISSUE THREAD

**Issue by robhyrk** [zeitgeistpm/ui#523]
[Performance Analysis](https://www.notion.so/zeitgeistpm/Performance-analysis-af50da6ec6ff475396d26f60de9aaf3f)

 _Stop waiting for rpc connection in order to start loading data for the app, use Subsquid instead. This is by far the biggest bottle neck, but will require SDK changes to decouple the rpc connection and graph ql client. This will benefit all pages on the app_

Reference to #255

**Comment by TvrtkoM** [comment]
this is already done with usage of `sdk-next` library and we are no longer waiting on rpc connection to start getting the data
